test/component-test.js

```diff
@@ -41,6 +41,16 @@ describe('component creation', () => {
       expect(content[0].props).to.eql({'data-prop': 'wat'});
       expect(content[0].content).to.eql(['hahaha']);
     });
+
+    it('accepts 3rd argument used as transform for each application', () => {
```

Comment:
What's an "application" in this context? An instance of "applying" data to a template? "Application" is such a programming domain word that it might be misleading here. Maybe not if React uses it in that way (but maybe it still is).